android/engine/src/main/java/org/smartregister/fhircore/engine/auth/AccountAuthenticator.kt

```diff
@@ -170,6 +174,18 @@ constructor(
 
   fun getUserInfo(): Call<ResponseBody> = oAuthService.userInfo()
 
+  // TODO move to some external file
+  suspend fun getPractitionerDetails(keycloak_uuid: String): org.hl7.fhir.r4.model.Bundle {
+
+    val iParser: IParser = FhirContextUtil().getPractitionerDetailParser()!!
+
+    val qJson =
+      context.assets.open("sample_practitionar_payload.json").bufferedReader().use { it.readText() }
+
+    return iParser.parseResource(qJson) as org.hl7.fhir.r4.model.Bundle
+  }
+  //    fhirResourceService.getResource("practitioner-details/$keycloak_uuid")
```

Comment:
remove

Reply:
Added the proper implementation since it has been deployed